Quick Start Documentation/Instrument Addon Quick Start Guide.md

```diff
@@ -0,0 +1,290 @@
+Note: this goes around all the VeriStand channels. They will not update as part of this.<br/>
+This provides a place to test and debug the instrument for things like basic connection, termination characters, message format​, etc.
+
+![Instrument Tool Command](./Images/image035.png)
+
+## Host Automation API
+
+In addition to the above items, there is an automation API that can be used from Windows to initiate commands with the add-on. This is installed at:<br/>
+C:\Users\Public\Documents\National Instruments\NI VeriStand <Year>\Custom Devices\Instrument Addon\Windows\Host Automation API<br/><br/>
+An example of using this API is as follows:
+
+![Host API Example](./Images/image036.png)
+
+First, connect to the add-on by its name and expected target IP address. Then send raw messages. When finished, close the connection.
+
+## System Definition API
+
+There is also a system definition API that allows the user to programatically create an instance of the add-on from a previously exported configuration.<br/>
+This is installed at:<br/>
+C:\Users\Public\Documents\National Instruments\NI VeriStand <Year>\Custom Devices\Instrument Addon\Windows\System Definition API<br/><br/>
+An example of using this API is as follows:​​​
+
+![System Definition API Example](./Images/image038.jpg)
+
+## Scripting API
+
+Users can add flexibility and reusability to their systems by creating and configuring system definitions through the Scripting API.
+
+## With this API, users can:<br/>
+- create a new Instrument from a previously exported configuration:
+
+    Node used: **Create Instrument Addon From Configuration File**
+
+![Create Instrument Addon From Configuration File](./Images/image040.png)
+
+- create a new Instrument from scratch<br/>
+    Node used: **Create Instrument Addon**
+
+![Create Instrument Addon](./Images/image041.png)
+
+- add Messages and configure their Command/Response 
+
+    Node used: **Create Message Group**<br/>
+![Create Message Group](./Images/image042.png)
+
+    Node used: **Add Command**<br/>
+![Add Command](./Images/image043.png)
+
+    Node used: **Add Response**<br/>
+![Add Response](./Images/image044.png)
+
+    Node used: **Create Message**<br/>
+
+![Create Message](./Images/image045.png)
+
+- Create Channels
+
+    Node used: **Create Global MetaData Channel**<br/>
+![Create Global MetaData Channel](./Images/image046.png)
+
+    Node used: **Create MetaData Channel**<br/>   
+![Create MetaData Channel](./Images/image047.png)
+
+    Node used: **Create Data Channel**<br/>
+![Create Data Channel](./Images/image048.png)
+
+- Fetch the Protocol object for a specific Instrument Addon
+
+    Node used: **Get Protocol**
+   
+- Change the Protocol object (all or specific properties)
+
+    Node used: **Set Protocol**   
+
+![Get and Set Protocol](./Images/image049.png)
+
+- Get and Set the PCL Decimation
+
+    Node used: **Get PCL Decimation**
+    Node used: **Set PCL Decimation**   
+
```

Comment:
This rendering does not look consistent.

![image](https://user-images.githubusercontent.com/62251693/94681469-2cccdd80-032c-11eb-843a-704adecf845e.png)

We can change the description to something like: "Nodes used: **Get PCL Decimation** and **Set PCL Decimation**" and do the same thing the Get Protocol and Set Protocol.